src/pageComponents/search/Results.jsx

```diff
@@ -69,8 +75,13 @@ class Results extends PureComponent {
     const { results } = this.props;
     return results.map((result, i) => {
       const { _index, _source: { title, url }, formattedDescription } = result;
+      const pathname = result._source.url;
```

Comment:
Same as `url` variable used on line 84. Remove and replace all instances of `pathname` with `url`

Reply:
Also, `pathname` would be a misnomer. As this can either be a relative link or an external URL.

To make it clear, `url` has been de-structured on the line above this declaration. and will hold the same value as your `pathname`